sw/device/silicon_creator/rom/e2e/BUILD

```
@@ -4,6 +4,8 @@
 
 load(
     "//rules:opentitan_test.bzl",
+    "DEFAULT_TEST_SUCCESS_MSG",
+    "ROM_BOOT_FAILURE_MSG",
```

If you agree with my comment above:
```suggestion
    "DEFAULT_TEST_FAILURE_MSG",
```

---

fixed.